templates/terraform/examples/legacy_spanner_database_ddl.erb

```diff
@@ -0,0 +1,13 @@
+resource "google_spanner_instance" "main" {
```

Comment:
One last thing - the reason it's failing to run the integration tests is that this file needs to be named 
templates/terraform/examples/legacy_spanner_database_ddl.tf.erb

The extension is just `.erb` right now and it needs to be `.tf.erb`.